packages/pharos/src/components/input-group/pharos-input-group.ts

```diff
@@ -44,14 +44,15 @@ export class PharosInputGroup extends PharosTextInput {
     super.updated(changedProperties);
 
     if (changedProperties.has('_prependGroupWidth')) {
-      super[
-        '_input'
-      ].style.paddingLeft = `calc(${PharosSpacingThreeQuartersX} + ${this._prependGroupWidth}px)`;
+      this._input.style.paddingLeft = `calc(${PharosSpacingThreeQuartersX} + ${this._prependGroupWidth}px)`;
     }
     if (changedProperties.has('_appendGroupWidth')) {
-      super[
-        '_input'
-      ].style.paddingRight = `calc(${PharosSpacingThreeQuartersX} + ${this._appendGroupWidth}px)`;
+      if (this._inputIcon) {
```

Comment:
Noticed the icon does not get placed correctly when `validated` and `invalidated` are added dynamically with the dev tools. We should probably react when either properties are true as well.